### DISCUSSION TITLE
Icon Request: fa-groucho

### DISCUSSION THREAD

**Discussion by nmorehart** [FortAwesome/Font-Awesome#21195] [Icon Requests]
I was looking for a good icon to indicate alias and my first thought was disguise. So, then I was looking for the groucho glasses/eyebrows/moustache because I thought that would be awesome.

Thank you!

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

This request is a duplicate of #5312, please add a 👍  reaction to that request.

Please make sure you [search](https://github.com/FortAwesome/Font-Awesome/search?type=Issues) before opening new requests

Closing here

**Comment by nmorehart** [comment]
I wouldn't say a plain moustache is the same as my request for something more "groucho."

[like this](http://img06.deviantart.net/2244/i/2012/092/3/9/groucho_glasses_by_mike44nh-d4ut2c6.png)

**Comment by nmorehart** [comment]
I responded but see you already closed my request.

[image: Inline image 1]

That isn't simply a moustache and I don't think it's a duplicate. I did
search on everything I could think of.

But you just closed my request because there's a moustache involved?

**Comment by tagliala** [comment]
@nmorehart sorry, got it

![image](https://cloud.githubusercontent.com/assets/556268/23141889/40e39fca-f7b9-11e6-97b2-6dd6fe01df8e.png)

Reopening

**Comment by nmorehart** [comment]
Thank you!

**Comment by nmorehart** [comment]
Hello there! I was wondering if this had made the repository somewhere. I am unable to locate it.

**Comment by tagliala** [comment]
Hi @nmorehart

nope, this is still a request

**Comment by nmorehart** [comment]
Thank you for the quick response!

**Comment by jarvism** [comment]
Don't forget an fa-un-groucho as well.

**Comment by aethior** [comment]
Hi,

Something like https://fontawesome.com/icons/face-disguise?

Regards.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
